### DISCUSSION TITLE
请问一下smartdns.log中的域名为host_ip_address是怎么来的

### DISCUSSION THREAD

**Discussion by huoshan12345** [pymumu/smartdns#1821] [Q&A]
![image](https://github.com/user-attachments/assets/479ea5b8-b528-4fd5-8695-b32905b696d2)

查询的域名里有很多的`host_ip_address`和`host_ip_address.lan`
请问一下这两个域名是代表什么呢

PS: 
1. 这两个域名解析出来的ip实际上就是我配置的几个上游dns
2. 它俩不在里面`/etc/hosts`里面，hosts内容如下：

```
127.0.0.1 localhost
192.168.8.1 oprx.lan local.oprx.lan
::1     localhost ip6-localhost ip6-loopback
ff02::1 ip6-allnodes
ff02::2 ip6-allrouters
```

完整的smartdns.log: 
[smartdns.log-20240916-222418.gz](https://github.com/user-attachments/files/17014817/smartdns.log-20240916-222418.gz)

**Comment by PikuZheng** [comment]
.lan常见于openwrt的dhcp server为本地主机附近的dns后缀，猜测是有个计算机名为host_ip_address。
我自己是阻断所有以.lan结尾的查询，因为没有用

**Reply by huoshan12345** [reply]
谢谢，.lan后缀貌似是dnsmasq里面配置的。

**Comment by huoshan12345** [comment]
是smaba这个程序一直在请求。
我用了土办法最后找到了，我遍历了openwrt系统盘上的所有文本文件，然后在/etc/samba/smb.conf.template里面找到了
